docs/content/docs/guide/gitops_commands.md

```diff
@@ -102,6 +104,81 @@ Please note that this feature is supported for the GitHub provider only.
 
 The PipelineRun will be restarted regardless of the annotations if the comment `/test <pipelinerun-name>` or `/retest <pipelinerun-name>` is used. This allows you to have control of PipelineRuns that get only triggered by a comment on the Pull Request.
 
+### Triggering PipelineRun on Git tags
+
+{{< support_matrix github_app="true" github_webhook="true" gitea="false" gitlab="false" bitbucket_cloud="false" bitbucket_server="false" >}}
+
+You can retrigger a PipelineRun against a specific Git tag by commenting on
+the tagged commit using a GitOps command. Pipelines-as-Code will resolve the
+tag to its commit SHA and run the matching PipelineRun against that commit.
+
+Supported commands:
+
+- `/test tag:<tag>`: retrigger all matching PipelineRuns for the tag commit
+- `/test <pipelinerun-name> tag:<tag>`: retrigger only the named PipelineRun
+- `/retest tag:<tag>`: retrigger all matching PipelineRuns for the tag commit
+- `/retest <pipelinerun-name> tag:<tag>`: retrigger only the named PipelineRun
+- `/cancel tag:<tag>`: cancel all running PipelineRuns for the tag commit
+- `/cancel <pipelinerun-name> tag:<tag>`: cancel only the named PipelineRun
+
+Examples:
+
+```text
+/test tag:v1.0.0
+```
+
+or
+
+```text
+/retest tag:v1.0.0
+```
+
+```text
+/cancel tag:v1.0.0
+```
+
+```text
+/cancel pipelinerun-on-tag tag:v1.0.0
+```
+
+```text
+/test pipelinerun-on-tag tag:v1.0.0
+```
+
+Notes:
+
+- The event type is treated as `push`; configure your PipelineRun with
+  `pipelinesascode.tekton.dev/on-event: "[push]"`.
+- This is currently supported on GitHub (App and Webhook) only.
+
+Minimal PipelineRun example:
+
+```yaml
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: pipelinerun-on-tag
+  annotations:
+    pipelinesascode.tekton.dev/on-target-branch: "[refs/tags/*]"
+    pipelinesascode.tekton.dev/on-event: "[push]"
+spec:
+  pipelineSpec:
+    tasks:
+      - name: tag-task
+        taskSpec:
+          steps:
+            - name: echo
+              image: registry.access.redhat.com/ubi9/ubi-micro
+              script: |
+                echo "tag: {{ git_tag }}"
+```
+
+How to comment on a tag commit (GitHub):
+
+1. Go to your repository and open the Tags view (or Releases).
+2. Click the tag (for example, `v1.0.0`) to navigate to its commit.
+3. Add a comment on the commit with one of the commands above.
```

Comment:
Is it possible to give the screenshot as well that will help users to understand more clearly